docs/en/serverless/quickstarts/collect-data-with-native-otlp.asciidoc

```diff
@@ -0,0 +1,159 @@
+
+The Managed OTLP Endpoint is a fully managed offering exclusively for Elastic Cloud users (initially available in Elastic Cloud Serverless only) that simplifies OpenTelemetry data ingestion. It provides an endpoint for OpenTelemetry SDKs and Collectors to send telemetry data, with Elastic handling scaling, data processing, and storage.
+
+This endpoint is designed for the following use cases:
+
+* Logs & Infrastructure Monitoring: Logs forwarded in OTLP format and host and Kubernetes metrics in OTLP format.
+* APM: Application telemetry in OTLP format.
+
+[%collapsible]
+.Differences from the existing Elastic APM Endpoint
+====
+The Elastic Cloud Managed OTLP Endpoint ensures that OpenTelemetry data is stored without any schema translation, preserving both OpenTelemetry semantic conventions and resource attributes. It supports ingesting OTLP logs, metrics, and traces in a unified manner, ensuring consistent treatment across all telemetry data. This marks a significant improvement over the {observability-guide}/apm-open-telemetry.html[existing functionality], which primarily focuses on traces and the APM use case.
+====
+
+[discrete]
+== Prerequisites
+
+* An {obs-serverless} project. To learn more, refer to <<observability-create-an-observability-project>>.
+* A system forwarding logs, metrics, or traces in OTLP (any OTel Collector or SDK—EDOT or community).
+
+[discrete]
+=== Limitations
+
+* The OTLP endpoint only supports histograms with delta temporality. Cumulative histograms are dropped.
+* Latency distributions based on histogram values have limited precision due to the fixed boundaries of explicit bucket histograms.
+* Per-project rate limits apply. If you hit this limit, reach out to our https://support.elastic.co[support team].
+
+[discrete]
+== Get started
+
+[discrete]
+=== Get your native OTLP endpoint credentials
+
+. <<observability-create-an-observability-project,Create a new {obs-serverless} project>>, or open an existing one.
+
+. In your {obs-serverless} project, go to **Add Data**.
+
+. Under **What do you want to monitor?** select **Application**, and then under **Monitor your application using** select **OpenTelemetry**.
++
+NOTE: Follow this flow for all use cases, including logs and infrastructure monitoring.
+
+. Copy the `OTEL_EXPORTER_OTLP_ENDPOINT` URL. Replace `.apm` with `.ingest` and save this value for later.
+
+[discrete]
+=== Create an API key
+
+. Click "Create an API Key" to generate a new API key. Copy this value for later.
+. (Optional) Test your new API key. You can do this by sending an empty JSON object to the `/v1/traces` endpoint. For example:
++
+[source,bash]
+----
+curl -X POST \
+  -H "Content-Type: application/json" \
+  -H "Authorization: ApiKey <api-key>" \
+  https://{YOUR_CLUSTER}.ingest.us-east-1.aws.elastic.cloud:443/v1/traces \
+  -d '{}'
+----
++
+The response should be similar to:
++
+[source,txt]
+----
+{"partialSuccess":{}}%  
```

Comment:
This is the result I got when testing this. Does this sound right?

Reply:
Looks good.